manager/integration/tests/test_basic.py

```diff
@@ -5047,7 +5048,41 @@ def test_delete_backup_during_restoring_volume():  # NOQA
        the status of volume restore condition is "False",
        the reason of volume restore condition is "RestoreFailure"
     """
-    pass
+    # Given create volume v1 and attach to a node
+    vol_v1_name = "vol-v1"
+    vol_v1 = create_and_check_volume(client, vol_v1_name, size=str(512 * Mi))
+    vol_v1.attach(hostId=get_self_host_id())
+    vol_v1 = wait_for_volume_healthy(client, vol_v1_name)
+
+    #  And write data 150M to volume v1
+    vol_v1_endpoint = get_volume_endpoint(vol_v1)
+    write_volume_dev_random_mb_data(vol_v1_endpoint, 1, 150)
+
+    # When create a backup of volume v1
+    #  And wait for that backup is completed
+    bv, b, snap2, data = create_backup(client, vol_v1_name)
+
+    #  And restore a volume v2 from volume v1 backup
+    vol_v2_name = "vol-v2"
+    client.create_volume(name=vol_v2_name,
+                         size=str(512 * Mi),
+                         numberOfReplicas=3,
+                         fromBackup=b.url)
+
+    #  And delete the backup immediately
+    delete_backup(client, bv.name, b.name)
+    volume = wait_for_volume_status(client, vol_v1_name,
+                                    "lastBackup", "")
+    assert volume.lastBackupAt == ""
+
+    #  Then volume v2 "robustness" should be "faulted"
+    #   And `status` of volume restore condition should be "False",
+    #   And `reason` of volume restore condition should be "RestoreFailure"
```

Comment:
Just wondering if those extra spaces in comments were need

Reply:
I just want to implement the test step with the BDD description. 
There is 2 spaces and 3 spaces after #.  That is not the rule from the testing method. 
I am used to using this format.

Reply:
Suggest also updating the test steps to BDD style. 

I think it's fine keeping the spacing for alignment in the test steps. But maybe we don't need the spacing in the comments for tracking the steps. WDYT @roger-ryao @chriscchien ?

Reply:
> Suggest also updating the test steps to BDD style.
> 
> I think it's fine keeping the spacing for alignment in the test steps. But maybe we don't need the spacing in the comments for tracking the steps. WDYT @roger-ryao @chriscchien ?

Hi @c3y1huang 
I could refer to [your previous PR](https://github.com/longhorn/longhorn-tests/pull/1124/files#diff-a8f0fde79c3b8e81457325c6c5f55d814da023c4116cacbfc5a82f42158911d4R497-R523). I think I should remove the spacing in the comments and update the test steps to BDD style.

cc @chriscchien

Reply:
After discussed with @longhorn/qa and @c3y1huang , I update the [PR](https://github.com/longhorn/longhorn-tests/compare/e48efa15cca8d5d3c1cc6276f42623115bf66b2b..2928ae8c5e1e2691ab6cb021a563efc17b7d1b71)